src/format.ts

```diff
@@ -34,6 +34,7 @@ enum Design {
 	PhotoEssay,
 	PrintShop,
 	Obituary,
+	Corrections,
```

Comment:
The only thing I'd add here is that whilst it might not be needed necessarily at the Scala client level, there'll need to be an update to both the Scala content client and by extension the FAPI client to include this new type. 

It's annoying but it's best to keep them in sync for now until the unified upstream type stuff using Thrift can come into play

Reply:
Thanks Josh! Do you know who the best person is to chat to about those changes?

Reply:
Probably me... Though I'm leaving in a couple of days. There are some docs in the Dotcom shared drive about it. The FAPI change is something for the Dotcom team to do, so don't worry about that. It's just a case of bumping the content client version in the FAPI client and then bumping both the content client and FAPI client in Frontend.

The content client change itself is something along these lines: https://github.com/guardian/content-api-scala-client/pull/333

General steps are: 
1. Go into the `Design` (or other type) bit of the CAPI Enrichment and add the condition for the new design to the predicates list. _Note: Things earlier in the list get higher priority so make sure you consider what should take priority when placing it._
2. Add a test for the new type. If there's a specific condition where it's likely it could be one of two types and this one should take priority, add a test for that too e.g.  [this example here](https://github.com/guardian/content-api-scala-client/pull/333/files#diff-bceb6955b79f9aba1261c36f00d8a15fa39c19a6af1edca54267cd7bf4343e32R406-R414)
3. Do a release of the content client. You'll need a CAPI key in order to do this because it runs tests during the release process. CAPI team can help with this.

Let me know if you want any help with it. If you make the change today or tomorrow I should be free to review/pair on it.

Reply:
Great stuff, thanks!